app/src/androidTest/java/com/lightricks/feedexercise/data/ViewModelTest.kt

```diff
@@ -9,38 +9,86 @@ import com.lightricks.feedexercise.database.UserProject
 import com.lightricks.feedexercise.ui.feed.FeedViewModel
 import com.lightricks.feedexercise.ui.feed.FeedViewModelFactory
 import org.junit.Assert
-import org.junit.Before
-import org.junit.Test
 import org.junit.runner.RunWith
-import java.util.concurrent.CountDownLatch
-import java.util.concurrent.TimeUnit
 
 @RunWith(AndroidJUnit4::class)
-class ViewModelTest (private var db : FeedDatabase,
-                     private var feedRepository: FeedRepository,
-                     private var userProjectList : List<UserProject>){
+class ViewModelTest(
```

Comment:
Why are there both `ViewModelTest` and `CallViewModelTests`?

Reply:
so that i can pass params to the test to make it more generic(such as different mock FeedApiServices), thats how I did it in the first exercise, is there a better way to do it?